src/components/picker/index.js

```diff
@@ -321,10 +323,12 @@ class Picker extends BaseComponent {
     }
 
     const textInputProps = TextField.extractOwnProps(this.props);
+    const style = textInputProps.style;
     const label = this.getLabel();
     return (
       <TextField
         {...textInputProps}
+        style={[styles.textField, style]}
```

Comment:
instead of using style for coloring the inner TextField 
I would have use `TextField`'s `color` prop.
this way we can then override it in our private code in our `Picker.config`

Reply:
Fixed